contracts/zero-ex/contracts/src/features/native_orders/NativeOrdersInfo.sol

```diff
@@ -389,4 +391,23 @@ abstract contract NativeOrdersInfo is
             uint256(params.orderTakerAmount)
         ).safeDowncastToUint128();
     }
+
+    /// @dev checks if a given address is registered to sign on behalf of a maker address
+    /// @param maker the maker address encoded in an order (can be a contract)
+    /// @param signer the address that is providing a signature (an EOA)
+    function isValidSigner(
+        address maker,
+        address signer
+    )
+        public
+        view
+        returns (bool)
+    {
+        // returns false if it the mapping doesn't exist
+        // returns the bool `allowed` if the mapping exists
```

Comment:
I think this comment is more cognitive overhead than just looking at the code.
```suggestion
```

Reply:
I agree on the second, but the first seems nonobvious to me